perf/istio-install/setup_istio.sh

```diff
@@ -142,7 +142,7 @@ function install_istio_with_helm() {
 function install_istio_with_istioctl() {
   local CR_PATH="${WD}/istioctl_profiles/${CR_FILENAME}"
   pushd "${ISTIOCTL_PATH}"
-  ./istioctl install --charts ../manifest apply -f "${CR_PATH}" --set "${SET_OVERLAY}" "${EXTRA_ARGS}"
```

Comment:
is this the correct path? if you're in bin, you need to go back out to the root dir, so i think it would be ../manifests

Reply:
yes, it should be the correct path, since we copy all files to tmp/ dir, the current path actually is the tmp dir.